designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/menu/DropDownMenu.kt

```diff
@@ -0,0 +1,46 @@
+package org.hisp.dhis.mobile.ui.designsystem.component.menu
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.hisp.dhis.mobile.ui.designsystem.theme.Radius
+import org.hisp.dhis.mobile.ui.designsystem.theme.SurfaceColor
+
+@Composable
+fun <T> DropDownMenu(
+    modifier: Modifier = Modifier,
+    items: List<MenuItemData<T>>,
+    expanded: Boolean = false,
+    selectedItemIndex: Int? = null,
+    onDismissRequest: () -> Unit,
+    onItemClick: (T) -> Unit,
+) {
+    MaterialTheme(shapes = MaterialTheme.shapes.copy(extraSmall = RoundedCornerShape(Radius.S))) {
```

Comment:
Do you think you could use our DHIS2 theme? we should also verify that the ripple color is following the theme colors too